spec/formtastic_integration_spec.rb

```diff
@@ -0,0 +1,49 @@
+# rubocop:disable AlignArray
```

Comment:
Because I used different indentation for building up the expected_html string, this way you can see clearly the html markup the addon has to provide.